docs/swagger.yaml

```diff
@@ -51,20 +66,20 @@ paths:
           schema:
             type: "array"
             items:
-              $ref: "#/definitions/FieldErrorDto"
+              $ref: "#/definitions/Error"
         401:
           description: "Unauthorized"
         500:
           description: "Internal Server Error"
           schema:
             type: "array"
             items:
-              $ref: "#/definitions/FieldErrorDto"
+              $ref: "#/definitions/Error"
```

Comment:
Not always returning error on 500.

Reply:
Can we _try_ to format everything as `Error`? It would be very nice if all errors could be parsed the same way.